azDevOps/azure/templates/v2/steps/deploy-k8s-app-kubectl-generic.yml

```diff
@@ -21,24 +22,44 @@ steps:
       login_azure: ${{ parameters.login_azure }}
       login_kubernetes: ${{ parameters.login_kubernetes }}
 
+  - bash: apt install jq
+    displayName: install jq
```

Comment:
We have containers for the builds, if `jq` is needed a PR to the `stacks-webapp-template` repo should be done to ensure it's included in the right image (`ci-k8s` iirc`)

Reply:
Here is the Dockerfile: https://github.com/amido/stacks-webapp-template/blob/master/libs/images/k8s-deploy/Dockerfile